app/src/main/java/com/ad340/whatdo/ToDoItemRecyclerViewAdapter.java

```diff
@@ -17,11 +17,10 @@
 import java.util.List;
 
 public class ToDoItemRecyclerViewAdapter extends RecyclerView.Adapter<ToDoItemRecyclerViewAdapter.ToDoItemViewHolder> {
-    private List<ToDoItem> toDoList;
     private Context context;
+    private List<Todo> todos;
 
-    ToDoItemRecyclerViewAdapter(List<ToDoItem> toDoList, Context context) {
-        this.toDoList = toDoList;
+    ToDoItemRecyclerViewAdapter(Context context) {
```

Comment:
Why don't we initialize the todo list here and then we can get rid of all of the null checks?

Reply:
The setTodos will still have to be called to monitor and update. So it would be initialized in 2 places essentially. It would just be a wash in how many lines in/out, and the null checks seem safer.